frmts/mrf/PNG_band.cpp

```diff
@@ -309,8 +309,12 @@ CPLErr PNG_Codec::CompressPNG(buf_mgr &dst, const buf_mgr &src)
     // Let the quality control the compression level
     // Start at level 1, level 0 means uncompressed
     int zlvl = img.quality / 10;
-    if (0 == zlvl)
+    if (1 > zlvl)
```

Comment:
maybe use ``std::clamp(img.quality / 10, 1, 9)`` for compactness and clarity ?

Reply:
Are we mandating C++17 now?

Reply:
> Are we requiring C++17 now?

yes, since 3.9: https://gdal.org/en/stable/development/rfc/rfc98_build_requirements_gdal_3_9.html